plasmapy/physics/transport.py

```diff
@@ -205,3 +209,1493 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
     ln_Lambda = ln_Lambda.to(units.dimensionless_unscaled).value
 
     return ln_Lambda
+
+
+@check_quantity({"T_e": {"units": units.K, "can_be_negative": False},
+                 "n_e": {"units": units.m**-3},
+                 "T_i": {"units": units.K, "can_be_negative": False},
+                 "n_i": {"units": units.m**-3},
+                 })
+def classical_transport(coeffs, T_e, n_e, T_i, n_i, ion_particle, Z=None,
+                        B=0.0, model='Braginskii',
+                        field_orientation='parallel',
+                        coulomb_log_ei=None, V_ei=None,
+                        coulomb_log_ii=None, V_ii=None,
+                        hall_e=None, hall_i=None,
+                        mu=None, theta=None):
+    r"""
+    Braginskii-esque classical transport coefficients
+
+    Notes
+    -----
+    Classical transport theory is derived by using kinetic theory to close the
+    plasma two-fluid (electron and ion fluid) equations in the collisional
+    limit. This function uses fitting functions from literature to calculate
+    the transport coefficients, which are the resistivity, thermoelectric
+    conductivity, thermal conductivity, and viscosity, which can be used
+    to close the two-fluid equations.
+
+    Note well the assumptions in the derivation of classical transport:
+    (1) velocity distribution function is close to Maxwellian (no extremely
+        strong gradients) which is indicated when the following conditions
```

Comment:
There's some sort of doc crash on this function's docstring related to indentation. I'll try to figure out where exactly it happens.